src/TwentyEightDayActiveUsersMetric.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace Tightenco\NovaGoogleAnalytics;
```

Comment:
These four classes are so similar I think I want us to extract them out to a base class that all four extend. LMK if you need any help with it.